include/SW_Files.h

```diff
@@ -51,25 +49,15 @@ typedef enum {
 	eEndFile
 } SW_FileIndex;
 
-
-/* =================================================== */
-/*            Externed Global Variables                */
-/* --------------------------------------------------- */
-extern char *InFiles[SW_NFILES];
-extern char _ProjDir[FILENAME_MAX];
-extern char weather_prefix[FILENAME_MAX];
-extern char output_prefix[FILENAME_MAX];
-
-
 /* =================================================== */
 /*             Global Function Declarations            */
 /* --------------------------------------------------- */
-void SW_F_read(const char *s);
-char *SW_F_name(SW_FileIndex i);
-void SW_F_construct(const char *firstfile);
-void SW_F_deconstruct(void);
-void SW_WeatherPrefix(char prefix[]);
-void SW_CSV_F_INIT(const char *s);
+void SW_F_read(LOG_INFO* LogInfo, PATH_INFO* PathInfo);
+char *SW_F_name(SW_FileIndex i, char *InFiles[]);
+void SW_F_construct(char *InFiles[], const char *firstfile, char _ProjDir[]);
+void SW_F_deconstruct(char *InFiles[]);
+void SW_WeatherPrefix(char prefix[], char weather_prefix[]);

```

Comment:
Same for `SW_WeatherPrefix()`: couldn't we just inline the one line of code for the one time it is used?